<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="57056" started-at="2014-11-22T14:57:32Z" finished-at="2014-11-22T14:58:29Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="57056" started-at="2014-11-22T14:57:32Z" finished-at="2014-11-22T14:58:29Z">
      <class name="TestCases.Home_Page">
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:TestCases.Home_Page@129cfbb]" name="setUp" is-config="true" duration-ms="28734" started-at="2014-11-22T20:27:32Z" finished-at="2014-11-22T20:28:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestCases.Home_Page.verifyHomeTabCal() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyHomeTabCal()[pri:0, instance:TestCases.Home_Page@129cfbb]" name="verifyHomeTabCal" duration-ms="26512" started-at="2014-11-22T20:28:01Z" finished-at="2014-11-22T20:28:27Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at TestCases.Home_Page.verifyHomeTabCal(Home_Page.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomeTabCal -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestCases.Home_Page@129cfbb]" name="tearDown" is-config="true" duration-ms="1712" started-at="2014-11-22T20:28:27Z" finished-at="2014-11-22T20:28:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.Home_Page -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
